RadixWallet/Features/ImportMnemonic/ImportMnemonic.swift

```diff
@@ -451,7 +452,10 @@ public struct ImportMnemonic: Sendable, FeatureReducer {
 			let toPaste = pasteboardClient.getString() ?? ""
 			return .send(.view(.debugMnemonicChanged(toPaste)))
 
-		case .debugUseTestingMnemonicWithActiveAccounts:
+		case .debugUseOlympiaTestingMnemonicWithActiveAccounts:
+			return .send(.view(.debugMnemonicChanged("section canoe half crystal crew balcony duty scout half robot avocado gas all effort piece")))
```

Comment:
I would make these local constants, but it's not important

Reply:
Might remove later...: